CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Fixed: `no-indistinguishable-colors` now ignores keyword color names within `url()`.
 - Fixed: string and verbose formatters no longer use an ambiguous colour scheme.
 - Fixed: string formatter no longer outputs an empty line if there are no problems.
+- Fixed: `no-descending-specificity` now ignores selectors ending on `,`
```

Comment:
Would it be better if instead of *ignoring* `.foo` in `.foo, {}` it just removed that trailing comma and still checked `.foo`?